source/classes.tex

```diff
@@ -1919,7 +1919,7 @@
 \pnum
 The implicitly-declared move assignment operator for a class \tcode{X} will have the form
 \begin{codeblock}
-X& X::operator=(X&&);
+X& X::operator=(X&&)
```

Comment:
This is awkward, as it now leaves the sentence unfinished.

Looking at the next paragraph, I see that there we use a _text_semicolon to structure the sentence. Do we have similar options here? Otherwise add a colon after "have the form"?

Reply:
This is also the case for [class.copy.ctor]p7 and p9, [class.copy.assign]p2, and [class.dtor]p3.

The idea of this change (and the request of the NB comment) is just to get to somewhere consistent. We should tackle any other changes / improvements here as separate editorial issues.